pkg/server/server.go

```diff
@@ -776,7 +776,7 @@ func NewServer(cfg Config, stopper *stop.Stopper) (serverctl.ServerStartupInterf
 			clock,
 			rangeFeedFactory,
 			keys.SpanConfigurationsTableID,
-			1<<20, /* 1 MB */
+			4<<20, /* 4 MB */
```

Comment:
@msbutler We should consider something similar for the span configuration streamer

Reply:
Actually, over there we use the rangefeedcache whose buffer is expressed in the number of rows, so we actually have a very large buffer currently.